packages/shared/apis/EthereumController.ts

```diff
@@ -48,15 +51,27 @@ export interface IEthereumController {
 
 @registerAPI('EthereumController')
 export class EthereumController extends RestrictedExposableAPI implements IEthereumController {
+  parcelIdentity = this.options.getAPIInstance(ParcelIdentity)
+
   @exposeMethod
   async requirePayment(toAddress: string, amount: number, currency: string): Promise<any> {
     await this.assertHasPermissions([PermissionItem.USE_WEB3_API])
+    await getUnityInstance().RequestWeb3ApiUse('requirePayment', {
+      toAddress,
+      amount,
+      currency,
+      sceneId: this.parcelIdentity.land?.sceneId || ''
+    })
     return requirePayment(toAddress, amount, currency)
   }
 
   @exposeMethod
   async signMessage(message: MessageDict) {
     await this.assertHasPermissions([PermissionItem.USE_WEB3_API])
+    await getUnityInstance().RequestWeb3ApiUse('signMessage', {
+      message: await messageToString(message),
+      sceneId: this.parcelIdentity.land?.sceneId || ''
```

Comment:
@leanmendoza do we have a standardized way to get the sceneId for both PX and regular scenes?